samples/relocate_vm.py

```diff
@@ -0,0 +1,202 @@
+                        help="name of the destination host")
+
+    parser.add_argument('-v', '--vm-name',
+                        required=True,
+                        help="name of the vm")
+    my_args = parser.parse_args()
+    return cli.prompt_for_password(my_args)
+
+
+def get_object(content, vimtype, name, disp=False):
+    """
+    Internal method to create objects of various vCenter related classes
+    :param content:
+    :param vimtype:
+    :param name:
+    :return: Object
+    """
+    obj = None
+    container = content.viewManager.CreateContainerView(content.rootFolder,
+                                                        vimtype,
+                                                        True)
+    for c in container.view:
+        if disp:
+            print("c.name:" + str(c.name))
+        if c.name == name:
+            obj = c
+            break
+    return obj
+
+
+def collect_template_disks(vm):
+    """
+        Internal method to collect template disks
+        :param vm: VM object
+        :return: list of template disks
+    """
+    template_disks = []
+    for device in vm.config.hardware.device:
+        if type(device).__name__ == "vim.vm.device.VirtualDisk":
+            datastore = device.backing.datastore
+            print("device.deviceInfo.summary:" + device.deviceInfo.summary)
+            print("datastore.summary.type:" + datastore.summary.type)
+            if hasattr(device.backing, 'fileName'):
+                disk_desc = str(device.backing.fileName)
+                print("Disc Discription -- {}".format(disk_desc))
+                drive = disk_desc.split("]")[0].replace("[", "")
+                print("drive:" + drive)
+                print("device.backing.fileName:" + device.backing.fileName)
+                template_disks.append(device)
+    return template_disks
+
+
+def construct_locator(template_disks, datastore_dest_id):
+    """
+        Internal method to construct locator for the disks
+        :param template_disks: list of template_disks
+        :param datastore_dest_id: ID of destination datastore
+        :return: locator
+    """
+    ds_disk = []
+    for index, wdisk in enumerate(template_disks):
+        print("relocate index:" + str(index))
+        print("disk:" + str(wdisk))
+        disk_desc = str(wdisk.backing.fileName)
+        drive = disk_desc.split("]")[0].replace("[", "")
+        print("drive:" + drive)
+        print("wdisk.backing.fileName:" + wdisk.backing.fileName)
+        locator = vim.vm.RelocateSpec.DiskLocator()
+        locator.diskBackingInfo = wdisk.backing
+        locator.diskId = int(wdisk.key)
+        locator.datastore = datastore_dest_id
+        ds_disk.append(locator)
+    return ds_disk
+
+
+def relocate_vm(vm_name, content, host_dest, datastore_dest=None, **kwargs):
+    """
+    This method relocates all vm's in the vm_list to the host_dest across
+    datacenters, clusters, datastores managed by a Vcenter
+
+    Args:
+        vm_list:
```

Comment:
Lets Make sample to work only for one VM. Change this to vm_name.
Change Help message